conf/carbon.conf.example

```diff
@@ -347,6 +353,11 @@ REPLICATION_FACTOR = 1
 # must be defined in this list
 DESTINATIONS = 127.0.0.1:2004
 
+# This define the protocol to use to contact the destination. It can be
+# set to one of "line", "pickle", "udp" and "protobuf". This list can be
+# extended with CarbonClientFactory plugins and defaults to "pickle".
+# DESTINATION_PROTOCOL = pickle
```

Comment:
Is that section needed? DESTINATION_PROTOCOL  is not used in files below.
And why the default is "pickle"?